jobs/jobscheduler.py

```diff
@@ -70,8 +71,23 @@ def runJob(job):
         job["exit_code"] = 0
     else:
         cmd = job["command"]
-        if ("gpu" in job) and (job["gpu"] >= 0):
-            cmd = cmd.replace("HIP_VISIBLE_DEVICES=0", "HIP_VISIBLE_DEVICES=%s" % job["gpu"])
+        if job.get("gpu", -1) >= 0:
+            maker, idx = job["gpu"].split(":", 1)
```

Comment:
if job["gpu"] is iof form `make:id` then I think the above condition ` job.get("gpu", -1) >= 0` will not work